internal/tools/go.sum

```diff
@@ -750,8 +750,8 @@ go.opentelemetry.io/build-tools/issuegenerator v0.7.0 h1:p4V6PhTQmp5kWBRqYf+bd6k
 go.opentelemetry.io/build-tools/issuegenerator v0.7.0/go.mod h1:dBqO2CqMnVm35etsxOYZqNtA+4ds5wGoELSAsau7itY=
 go.opentelemetry.io/build-tools/multimod v0.7.0 h1:Af1wVNaJRMVqRkiiRjeMrAfKJDejvpsLoBF9H5UrjSo=
 go.opentelemetry.io/build-tools/multimod v0.7.0/go.mod h1:WAwBtJC42JbRDzfi3erCYQ9M1RnyoIcqI/H8eqBBPQo=
-go.opentelemetry.io/collector/cmd/builder v0.75.0 h1:oAMeqGm4IfNeFKNXspmVCp9qK6CvBe1Ag+0hx4cAIZA=
-go.opentelemetry.io/collector/cmd/builder v0.75.0/go.mod h1:tIITwEHJUrMfMe13obMA9C81CqR4eiXB3unf5rZdXxE=
+go.opentelemetry.io/collector/cmd/builder v0.76.1 h1:CbKqIud0XAZuMcj4xyB5GDJkJ4xQsbN1yclox+wPIbM=
```

Comment:
Something is trying to pull in an older version of the builder?

Reply:
The builder lives in core and is 0.76.1.

Reply:
Didn't catch that, my visual grep was for anything not `v0.76.3`